notebooks/supernovae_plus_extended_source_tutorial.ipynb

```diff
@@ -68,7 +68,8 @@
     "# skypy config\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    sn_modeldir = "/Users/narayankhadka/downloads/sncosmo_sn_models/SALT3.NIR_WAVEEXT"
</pre><p>can you make some comments of where these parts need to be downloaded from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>